src/main/java/domain/Car.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.Random;
+
+public class Car {
+    public static final int INITIAL_POSITION = 1; //위치 기본값 상수
+    public static final int MOVE_VALUE = 4;//움직임 기준 상수
+    private final String name;
+    private int position;
```

Comment:
공백을 통해 상수와 필드를 분리해서 보여주면 좋습니다!
```suggestion
    public static final int INITIAL_POSITION = 1; //위치 기본값 상수
    public static final int MOVE_VALUE = 4;//움직임 기준 상수
    
    private final String name;
    private int position;
```